debian/marquee/nginx/conf/http.conf

```diff
@@ -0,0 +1,10 @@
+server {
+    listen 80 default_server;
+    listen [::]:80 default_server;
+    location ^~ /.well-known/acme-challenge/ {
+        default_type application/jose+json;
+    }
+    location / {
+        return 301 https://$host$request_uri;
+    }
```

Comment:
Oops, an important detail is missing here:
> root /var/www/http;

Had to put that on marquee to get a new cert.